<<<<<<< HEAD
CRITICAL - 2022-11-02 19:50:57 --> Call to undefined method App\Controllers\Tamaños::findAll()
in APPPATH\Controllers\Tamaños.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 19:53:43 --> Call to undefined method App\Controllers\Tamaños::findAll()
in APPPATH\Controllers\Tamaños.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 19:56:18 --> Call to undefined method App\Controllers\Tamaños::findAll()
in APPPATH\Controllers\Tamaños.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-02 20:05:03 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('2')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:05:24 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('2')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:12:35 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('2')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:13:35 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('2')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:15:21 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('1')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:15:54 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('1')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:27:54 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('1')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:28:07 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('1')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:33:36 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('1')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:33:47 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('2')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:33:52 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('3')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:41:44 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('3')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:43:12 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('3')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:45:11 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('3')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:47:49 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('3')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 20:48:07 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('3')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-02 21:04:49 --> Undefined variable $especialidad
in APPPATH\Controllers\Platos.php on line 48.
 1 APPPATH\Controllers\Platos.php(48): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $especialidad', 'APPPATH\\Controllers\\Platos.php', 48)
 2 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('3')
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-02 21:08:36 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('3')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:09:09 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:09:19 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('3')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:09:23 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('2')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:09:25 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('1')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:24:10 --> mysqli_sql_exception: Duplicate entry '7' for key 'PRIMARY' in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pl...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pl...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pl...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pl...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(666): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(22): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->guardarPlatos()
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:26:25 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:26:25 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:28:38 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:30:20 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:31:49 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:36:25 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:40:39 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:40:51 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:48:17 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:48:28 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 21:53:27 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('6')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 22:28:47 --> mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY' in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `me...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `me...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `me...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `me...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(666): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Meseros.php(24): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Meseros->guardarMeseros()
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Meseros))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-11-02 22:31:58 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`proyecto_final_pp_2022`.`platos`, CONSTRAINT `platos_ibfk_1` FOREIGN KEY (`id_tamaño`) REFERENCES `tamaños` (`id_tamaño`) ON UPDATE CASCADE) in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `platos`...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `platos`...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `platos`...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query('UPDATE `platos`...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(692): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Platos.php(50): CodeIgniter\Model->update(NULL, Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Platos->actualizarplatos('7')
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Platos))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-02 23:26:15 --> Call to undefined method App\Controllers\Tamaños::findAll()
in APPPATH\Controllers\Tamaños.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:32:38 --> Undefined variable $listaTamaño
in APPPATH\Views\tamaños.php on line 81.
 1 APPPATH\Views\tamaños.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $listaTamaño', 'APPPATH\\Views\\tamaños.php', 81)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tamaños.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('tamaños', [], true)
 5 APPPATH\Controllers\Tamaños.php(11): view('tamaños', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:34:20 --> Undefined variable $listaTamaño
in APPPATH\Views\tamaños.php on line 81.
 1 APPPATH\Views\tamaños.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $listaTamaño', 'APPPATH\\Views\\tamaños.php', 81)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tamaños.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('tamaños', [], true)
 5 APPPATH\Controllers\Tamaños.php(11): view('tamaños', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:35:30 --> Undefined variable $listaTamaño
in APPPATH\Views\tamaños.php on line 81.
 1 APPPATH\Views\tamaños.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $listaTamaño', 'APPPATH\\Views\\tamaños.php', 81)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tamaños.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('tamaños', [], true)
 5 APPPATH\Controllers\Tamaños.php(11): view('tamaños', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:36:15 --> Undefined variable $listaTamaño
in APPPATH\Views\tamaños.php on line 81.
 1 APPPATH\Views\tamaños.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $listaTamaño', 'APPPATH\\Views\\tamaños.php', 81)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tamaños.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('tamaños', [], true)
 5 APPPATH\Controllers\Tamaños.php(11): view('tamaños', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:37:41 --> Undefined variable $listaTamaño
in APPPATH\Views\tamaños.php on line 81.
 1 APPPATH\Views\tamaños.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $listaTamaño', 'APPPATH\\Views\\tamaños.php', 81)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tamaños.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('tamaños', [], true)
 5 APPPATH\Controllers\Tamaños.php(11): view('tamaños', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:39:45 --> Undefined variable $listaTamaño
in APPPATH\Views\tamaños.php on line 81.
 1 APPPATH\Views\tamaños.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $listaTamaño', 'APPPATH\\Views\\tamaños.php', 81)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tamaños.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('tamaños', [], true)
 5 APPPATH\Controllers\Tamaños.php(11): view('tamaños', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:44:07 --> Undefined variable $lista_Tamaño
in APPPATH\Views\tamaños.php on line 81.
 1 APPPATH\Views\tamaños.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $lista_Tamaño', 'APPPATH\\Views\\tamaños.php', 81)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tamaños.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('tamaños', [], true)
 5 APPPATH\Controllers\Tamaños.php(11): view('tamaños', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:44:13 --> Undefined variable $lista_Tamaño
in APPPATH\Views\tamaños.php on line 81.
 1 APPPATH\Views\tamaños.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $lista_Tamaño', 'APPPATH\\Views\\tamaños.php', 81)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tamaños.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('tamaños', [], true)
 5 APPPATH\Controllers\Tamaños.php(11): view('tamaños', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:44:40 --> Undefined variable $lista_Tamaño
in APPPATH\Views\tamaños.php on line 81.
 1 APPPATH\Views\tamaños.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $lista_Tamaño', 'APPPATH\\Views\\tamaños.php', 81)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tamaños.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('tamaños', [], true)
 5 APPPATH\Controllers\Tamaños.php(11): view('tamaños', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:49:19 --> Undefined variable $lista_Tamaño
in APPPATH\Views\tamaños.php on line 81.
 1 APPPATH\Views\tamaños.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $lista_Tamaño', 'APPPATH\\Views\\tamaños.php', 81)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tamaños.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('tamaños', [], true)
 5 APPPATH\Controllers\Tamaños.php(11): view('tamaños', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:50:58 --> Undefined variable $elemento_Tamaño
in APPPATH\Views\tamaños.php on line 87.
 1 APPPATH\Views\tamaños.php(87): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $elemento_Tamaño', 'APPPATH\\Views\\tamaños.php', 87)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tamaños.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('tamaños', [], true)
 5 APPPATH\Controllers\Tamaños.php(11): view('tamaños', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->buscarTamaños()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:56:48 --> Invalid file: actualizar_tamaño.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('actualizar_tamaño.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('actualizar_tamaño', [], true)
 3 APPPATH\Controllers\Tamaños.php(37): view('actualizar_tamaño', [...])
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->verTamaños('5,0')
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:56:48 --> Invalid file: actualizar_tamaño.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('actualizar_tamaño.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('actualizar_tamaño', [], true)
 3 APPPATH\Controllers\Tamaños.php(37): view('actualizar_tamaño', [...])
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Tamaños->verTamaños('5,0')
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tamaños))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
=======
CRITICAL - 2022-11-02 23:07:53 --> Invalid file: jefe.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('jefe.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('jefe', [], true)
 3 APPPATH\Controllers\Jefes.php(11): view('jefe', [...])
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Jefes->buscarJefes()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:11:13 --> Invalid file: jefe.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('jefe.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('jefe', [], true)
 3 APPPATH\Controllers\Jefes.php(11): view('jefe', [...])
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Jefes->buscarJefes()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:15:29 --> Invalid file: jefe.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('jefe.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('jefe', [], true)
 3 APPPATH\Controllers\Jefes.php(11): view('jefe', [...])
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Jefes->buscarJefes()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:17:25 --> Invalid file: jefe.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('jefe.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('jefe', [], true)
 3 APPPATH\Controllers\Jefes.php(11): view('jefe', [...])
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Jefes->buscarJefes()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:21:50 --> Invalid file: jefe.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('jefe.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('jefe', [], true)
 3 APPPATH\Controllers\Jefes.php(11): view('jefe', [...])
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Jefes->buscarJefes()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:23:47 --> Invalid file: jefe.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('jefe.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('jefe', [], true)
 3 APPPATH\Controllers\Jefes.php(11): view('jefe', [...])
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Jefes->buscarJefes()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:24:30 --> Undefined variable $lista_gerente
in APPPATH\Views\jefes.php on line 95.
 1 APPPATH\Views\jefes.php(95): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $lista_gerente', 'APPPATH\\Views\\jefes.php', 95)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\jefes.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('jefes', [], true)
 5 APPPATH\Controllers\Jefes.php(11): view('jefes', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Jefes->buscarJefes()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:26:14 --> Undefined variable $lista_gerente
in APPPATH\Views\jefes.php on line 95.
 1 APPPATH\Views\jefes.php(95): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $lista_gerente', 'APPPATH\\Views\\jefes.php', 95)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\jefes.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('jefes', [], true)
 5 APPPATH\Controllers\Jefes.php(11): view('jefes', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Jefes->buscarJefes()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:26:34 --> Undefined variable $elemento_cocinero
in APPPATH\Views\jefes.php on line 106.
 1 APPPATH\Views\jefes.php(106): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $elemento_cocinero', 'APPPATH\\Views\\jefes.php', 106)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\jefes.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('jefes', [], true)
 5 APPPATH\Controllers\Jefes.php(11): view('jefes', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Jefes->buscarJefes()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-02 23:31:32 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 846.
 1 SYSTEMPATH\BaseModel.php(846): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(692): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\Jefes.php(54): CodeIgniter\Model->update('1', [...])
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Jefes->actualizarJefes()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-02 23:42:40 --> mysqli_sql_exception: Column 'telefono' cannot be null in C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ProyectoFinal\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\ProyectoFinal\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\ProyectoFinal\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ProyectoFinal\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ProyectoFinal\system\Model.php(666): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ProyectoFinal\app\Controllers\Cocineros.php(24): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(898): App\Controllers\Cocineros->guardarCocineros()
#9 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cocineros))
#10 C:\xampp\htdocs\ProyectoFinal\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ProyectoFinal\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
<<<<<<< HEAD
CRITICAL - 2022-11-02 23:52:14 --> Trying to access array offset on value of type null
in APPPATH\Views\actualizar_jefes.php on line 18.
 1 APPPATH\Views\actualizar_jefes.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Views\\actualizar_jefes.php', 18)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\actualizar_jefes.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('actualizar_jefes', [], true)
 5 APPPATH\Controllers\Jefes.php(40): view('actualizar_jefes', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Jefes->verJefes('502,0')
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
=======
>>>>>>> b75bcc024da9981aa5d015b1d5c79d425a2d3ee0
>>>>>>> 46fbc176c19af31b33535bdaf3a6f101deca56a8
